staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -246,7 +259,42 @@ func (a *GenericAdmissionWebhook) Admit(attr admission.Attributes) error {
 	return errs[0]
 }
 
+// whether the namespace is exempted from the webhook.
+func (a *GenericAdmissionWebhook) exemptNamespace(h *v1alpha1.ExternalAdmissionHook, attr admission.Attributes) (bool, error) {
+	namespaceName := attr.GetNamespace()
+	if namespaceName == "" {
+		// if the request is root scoped, it is not exempted.
+		return false, nil
```

Comment:
This is a neat side-effect.  A downed admission plugin could prevent the node that would run the admission webhook from running.

I don't think we need to resolve it for beta, but given the switch to a DNS name, perhaps admission plugins that can mess with nodes should be hosted in a way that allows off cluster access.

Reply:
I think this can be solved with the run level concept, like by reserving some nodes for run level 0 items.